### DISCUSSION TITLE
pip missing when building own renovate Docker image

### DISCUSSION THREAD

**Discussion by sudoleg** [renovatebot/renovate#29992] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Hello renovate team,

we have problems with building our own Docker Image. We use the Dockerfile provided [here](https://docs.renovatebot.com/examples/self-hosting/#renovate-docker-image). However, when building the image, we get the following error:

```bash
INFO[0023] RUN pip config set global.cert /etc/ssl/certs/ca-certificates.crt 
INFO[0023] Cmd: /bin/bash                               
INFO[0023] Args: [-c pip config set global.cert /etc/ssl/certs/ca-certificates.crt] 
INFO[0023] Util.Lookup returned: &{Uid:1000 Gid:0 Username:ubuntu Name: HomeDir:/home/ubuntu} 
INFO[0023] Performing slow lookup of group ids for ubuntu 
INFO[0023] Running: [/bin/bash -c pip config set global.cert /etc/ssl/certs/ca-certificates.crt] 
/bin/bash: pip: command not found
error building image: error building stage: failed to execute command: waiting for process to exit: exit status 127
```

It appears as `pip` is not present in the renovate base image...

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment, marked as answer]
that docs seems to be outdated. skip the pip and node stuff and only set the openssl file. almost all tools should work out of the box

**Reply by sudoleg** [reply]
Thank you for the reply, I will submit a PR to fix the outdated docs :)

_Edit by HonkingGoose:_

We got the PR and merged it:

- #30096
